data/json/recipes/recipe_food.json

```diff
@@ -7693,6 +7693,18 @@
   ]
 },{
   "type" : "recipe",
+  "result": "vodka",
```

Comment:
I would distinguish between lab and food grade ethanol by lumping the former into "denatured alcohol" (i.e. ethanol rendered unfit for human consumption for cost/legal reasons). Someone familiar with lab-grade ethanol (from actually using it in a lab, for instance :)) could provide valuable information.